common/src/main/java/com/wynntils/models/activities/ActivityTrackerScoreboardPart.java

```diff
@@ -42,9 +44,20 @@ public void onSegmentChange(ScoreboardSegment newValue) {
         List<StyledText> taskLines = content.subList(1, content.size());
 
         for (StyledText line : taskLines) {
+            String unformatted = line.getString(PartStyle.StyleType.NONE);
+            Matcher spacerMatcher = SPACER_PATTERN.matcher(unformatted);
+            if (spacerMatcher.matches()) {
+                // There is a special character at the start of the line, we don't need the previous space
+                nextTask.deleteCharAt(nextTask.length() - 1);
```

Comment:
If the very first character is a special character, it will try to delete past the start of the string, If `deleteCharAt` just ignores a -1 argument then all is fine, otherwise this will be bad. (If sending -1 is okay, please add a comment stating that it is okay)